webapp/channels/src/components/admin_console/admin_console.test.tsx

```diff
@@ -102,8 +103,8 @@ describe('components/AdminConsole', () => {
             consoleAccess: {read: {}, write: {}},
             team: {name: 'development'} as Team,
         };
-        const wrapper = shallow(
-            <AdminConsole {...props}/>,
+        const wrapper = renderWithIntl(
+            <BrowserRouter><AdminConsole {...props}/></BrowserRouter>,
```

Comment:
Both of these tests, both in the new mono-repo and in the [old webapp](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/__snapshots__/admin_console.test.tsx.snap#L3-L12) have the exact same snapshot output even though they are supposed to be testing opposite scenarios (stay on page vs redirect because of insufficient permissions).

Because of that, they aren't testing anything and I'd advocate deleting them altogether. If we really feel strongly in the future on covering this, we will add propert tests then. I don't think its worth investing the set up in trying to make these meaningful, real tests now.

Reply:
@saturninoabril Thoughts on this.